libs/app-shell/src/lib/api/events/topics.service.ts

```diff
@@ -9,22 +9,22 @@ import { Injectable } from '@angular/core';
 export class MessagingTopics {
     public topicsDef = new Map<string, Topic>();
 
-    defineTopic(topic: Topic): void {
+    set(topic: Topic): void {
```

Comment:
Let's stick with define. We dont need call it defineTopic, but define should be there. 

Not even create, you are not creating this topic yet. you just creating definition of it.